services/app-web/src/pages/CMSDashboard/CMSDashboard.tsx

```diff
@@ -1,75 +1,40 @@
-import { GridContainer, Table, Tag } from '@trussworks/react-uswds'
-import classnames from 'classnames'
-import dayjs from 'dayjs'
+import { GridContainer } from '@trussworks/react-uswds'
 import React from 'react'
-import { NavLink } from 'react-router-dom'
 
 import { packageName } from '../../common-code/healthPlanFormDataType'
 import { base64ToDomain } from '../../common-code/proto/healthPlanFormDataProto'
-import { Loading } from '../../components/Loading'
-import {
-    SubmissionStatusRecord,
-    SubmissionTypeRecord,
-} from '../../constants/healthPlanPackages'
+import { SubmissionTypeRecord } from '../../constants/healthPlanPackages'
 import { useAuth } from '../../contexts/AuthContext'
-import {
-    HealthPlanPackageStatus,
-    Program,
-    useIndexHealthPlanPackagesQuery,
-} from '../../gen/gqlClient'
+import { useIndexHealthPlanPackagesQuery } from '../../gen/gqlClient'
 import { mostRecentDate } from '../../common-code/dateHelpers'
 import styles from '../StateDashboard/StateDashboard.module.scss'
 import { recordJSException } from '../../otelHelpers/tracingHelper'
 import {
     handleApolloError,
     isLikelyUserAuthError,
 } from '../../gqlHelpers/apolloErrors'
-import { ErrorAlertFailedRequest, ErrorAlertSignIn } from '../../components'
-
-// We only pull a subset of data out of the submission and revisions for display in Dashboard
-// Depending on submission status, CMS users look at data from current or previous revision
```

Comment:
todo: Please preserve this comment ^ somewhere. 

How `submittedAt` and `updatedAt` are calculated is different between cms and state user (since cms can't see unlocked submissions being edited or draft submissions). Want to make sure this is noted.